CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.0.1
+
+- Fix the task list template to use the correct status tag options
```

Comment:
Can this include the PR number and link, to match the format used in previous versions please?

Reply:
Done.